examples/cleanup-controller/deployment.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cleanup-controller
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cleanup
+  template:
+    metadata:
+        labels:
+          app: cleanup
+    spec:
+      serviceAccount: cleanup-controller
+      containers:
+      - name: cleanup-controller
+        image: gcr.io/justinblalock-joonix/local-volume-provisioner-cleanup:latest_linux_amd64
```

Comment:
I think you can substitute your gcr with what we plan for it to be: gcr.io/k8s-staging-sig-storage/local-volume-node-cleanup:canary

Reply:
Done